src/main/java/onboarding/Problem3.java

```diff
@@ -1,8 +1,41 @@
 package onboarding;
 
+import java.util.stream.IntStream;
+
 public class Problem3 {
+
     public static int solution(int number) {
-        int answer = 0;
-        return answer;
+        return new Solver3(number).solve();
+    }
+}
+
+final class Solver3 {
+
+    private final int limitNumber;
+    private int totalClaps = 0;
+
+    public Solver3(final int number) {
+        this.limitNumber = number;
+    }
+
+    public int solve() {
+        // endExclusive여서 +1 함
+        IntStream.range(1, limitNumber + 1).forEach(this::addClap);
+        return this.totalClaps;
+    }
+
+    /**
+     * 각 자릿수의 수가 3의 배수인 경우 손뼉을 1회씩 추가한다.
+     */
+    private void addClap(int x) {
+        int claps = 0;
+        int curDigit;
+
+        for (; x > 0; x /= 10) {
+            curDigit = x % 10;
+            if (curDigit == 0) continue;
```

Comment:
제가 for loop 에서 continue를 사용하는 것이 생소해서 질문드립니다!
```suggestion
            if (curDigit % 3 == 0 && curDigit != 0) claps++;
```
이런 식으로 변경하는 건 어떤가요?

Reply:
오 일단 저는 continue가 더 직관적이어서 이렇게 작성을 했습니다~ 저는 continue를 보면 생각을 덜 할 수 있는 것 같아서 좋다고 생각했는데요, 혹시 continue에 관한 규칙이 있을까요?

Reply:
제가 헷갈렸네요! 클린 코드 책에서 continue를 사용하지 마라는 구절을 본 것 같아서 리뷰를 달았습니다. 근데 다시 알아보니 작은 함수에서는 continue를 사용하는 것도 좋다고 얘기했었네요..! 그대로 두어도 괜찮을 것 같습니다. 제 불찰입니다 ㅜ

관련내용 링크 첨부 했습니다!
https://velog.io/@freesky/Clean-Code-%ED%95%A8%EC%88%98